.upptimerc.yml

```diff
@@ -1,36 +1,38 @@
-# Change these first
-owner: upptime # Your GitHub organization or username, where this repository lives
-repo: upptime # The name of this repository
+owner: lincbrain
+repo: linc-upptime
 
 sites:
-  - name: Google
-    url: https://www.google.com
-  - name: Wikipedia
-    url: https://en.wikipedia.org
-  - name: Hacker News
-    url: https://news.ycombinator.com
-  - name: Test Broken Site
-    url: https://thissitedoesnotexist.koj.co
-  - name: IPv6 test
-    url: forwardemail.net
-    port: 80
-    check: "tcp-ping"
-    ipv6: true
+  - name: Data portal
+    url: https://lincbrain.org
+    assignees:
+      - kabilar
+      - aaronkanzer
+  - name: DANDI Hub
+    url: https://hub.dandiarchive.org
+    assignees:
+      - kabilar
+      - aaronkanzer
+  - name: Documentation
+    url: https://docs.lincbrain.org
+    assignees:
+      - kabilar
+  - name: Dashboard
+    url: https://dashboard.lincbrain.org
+    assignees:
+      - kabilar
+  - name: Homepage
+    url: https://connects.mgh.harvard.edu
+    assignees:
+      - kabilar
 
 status-website:
-  # Add your custom domain name, or remove the `cname` line if you don't have a domain
-  # Uncomment the `baseUrl` line if you don't have a custom domain and add your repo name there
-  cname: demo.upptime.js.org
-  # baseUrl: /your-repo-name
-  logoUrl: https://raw.githubusercontent.com/upptime/upptime.js.org/master/static/img/icon.svg
-  name: Upptime
-  introTitle: "**Upptime** is the open-source uptime monitor and status page, powered entirely by GitHub."
-  introMessage: This is a sample status page which uses **real-time** data from our [GitHub repository](https://github.com/upptime/upptime). No server required — just GitHub Actions, Issues, and Pages. [**Get your own for free**](https://github.com/upptime/upptime)
+  cname: status.lincbrain.org
+  logoUrl: https://raw.githubusercontent.com/lincbrain/linc-artwork/refs/heads/main/linc.logo.color%2Bblack.alpha.notext.png
+  name: LINC Status
+  introTitle: "Status page of LINC services"
+  introMessage: This is a status page which uses **real-time** data from our [GitHub repository](https://github.com/lincbrain/linc-upptime).
   navbar:
     - title: Status
       href: /
     - title: GitHub
-      href: https://github.com/$OWNER/$REPO
-
-# Upptime also supports notifications, assigning issues, and more
-# See https://upptime.js.org/docs/configuration
+      href: https://github.com/$OWNER/$REPO
```

Comment:
Do we need to set these as env vars (or simply hard-code?) 

I know there are the two yaml values on the top of this file: `owner` and `repo` -- however, can we confirm that those get populated here?

Reply:
That's a good question.  Based on the upptime docs its not clear.  The variables are not defined in the `dandi/upptime` repo and it seems to work.

Reply:
Hmmm yeah it seems from https://upptime.js.org/docs/configuration/#repository you configured the supporting yaml correctly, but the docs also say that the $ENV are for secrets, so I assume perhaps they are stored elsewhere?

Perhaps just to move forward, hard-coding the values are fine, unless you are able to easily deploy and confirm/deny

Reply:
Yea, I am not entirely sure.  Will proceed with the deployment and see how it works.